### ISSUE TITLE
Skip namespaces?

### ISSUE THREAD

**Issue by michihenning** [lvc/abi-dumper#4]
When dumping an ABI with abi-compliance-checker, it is possible to exclude namespaces with the <skip-namespaces> directive. Would it be possible to do the same for abi-dumper? I'm getting lots of noise about added symbols in internal namespaces that have nothing to do with the public API. It would be nice to be able to suppress that.

**Comment by aponomarenko** [comment]
Yes, there is no such option yet in both tools. I'll add it soon in the abi-compliance-checker. As the temporary workaround you can use the `-skip-internal-symbols` and `-skip-internal-types` options of the abi-compliance-checker to skip checking of private symbols:

```
abi-compliance-checker ... -skip-internal-symbols "\d(NS1|NS2|...)" -skip-internal-types "(NS1|NS2|...)::"
```

where NS1, NS2, ... — private namespaces.
